src/ast/function.rs

```diff
@@ -59,7 +77,18 @@ pub enum Function {
     #[strum(to_string = "SQRT")]
     Sqrt(Expr),
     #[strum(to_string = "POWER")]
-    Power { expr: Expr, power: Expr },
+    Power {
+        expr: Expr,
+        power: Expr,
+    },
+    Ltrim {
+        expr: Expr,
+        chars: Option<Expr>,
+    },
+    Rtrim {
+        expr: Expr,
+        chars: Option<Expr>,
+    },
```

Comment:
Let's `#[strum(to_string = "LTRIM")]` and `#[strum(to_string = "RTRIM")]` to capitalize function names for the consistency.